Assets/Content/Furniture/Storage/Carts/Motorized Janitorial Cart.prefab

```diff
@@ -0,0 +1,229 @@
+%YAML 1.1
```

Comment:
When adding new files, please name them with PascalCase format for consistency.
So "Motorized Janitorial Cart.prefab" should be "MotorizedJanitorialCart.prefab"